Serhii_Oliinyck/Bob.py

```diff
@@ -0,0 +1,26 @@
+#-------------------------------------------------------------------------------
+# Name:        module1
+# Purpose:
+#
+# Author:      soliinyk
+#
+# Created:     23/10/2018
+# Copyright:   (c) soliinyk 2018
+# Licence:     <your licence>
+#-------------------------------------------------------------------------------
+
+def main(string):
+    if(string == ""):
+        return "Fine. Be that way!"
+    elif(string.isupper() and (string[len(string) - 1] == "?")):
```

Comment:
There's a method endswith() or you can also use [-1] index to get the last element of the sequence